extensions/src/ACRE2Arma/common/directx/d3d_display.cpp

```diff
@@ -227,7 +227,7 @@ namespace acre {
             // Create window
             _hInst = (HINSTANCE)GetCurrentProcess();
             RECT rc = { 0, 0, (LONG)width, (LONG)height };
-            AdjustWindowRect(&rc, WS_OVERLAPPEDWINDOW, FALSE);
+            AdjustWindowRect(&rc, WS_OVERLAPPEDWINDOW, false);
```

Comment:
It has BOOL as a parameter, therefore  it should be FALSE
```suggestion
            AdjustWindowRect(&rc, WS_OVERLAPPEDWINDOW, FALSE);
```